src/api/controllers/misc.ts

```diff
@@ -155,3 +157,73 @@ export function handleInternalWebhooksUpdate(data: DiscordPayload) {
     options.guild_id,
   );
 }
+
+export function handleInternalIntegrationCreate(
+  data: DiscordPayload,
+) {
+  if (data.t !== "INTEGRATION_CREATE") return;
+
+  const {
```

Comment:
Use the snake case stuff for these now

Reply:
It should be fine to use camel case for the data supplied to the user for now, once the "snake case to camel case" PR is merged, we can use it to convert the properties into camel case easily.